src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,16 @@
 
 public class MainApp {
 
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Bulldozer bulldozer = new Bulldozer();
+        Excavator excavator = new Excavator();
```

Comment:
[Use abstract reference instead of specific one where possible](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/oop)

Reply:
done